src/openforms/js/components/admin/form_design/logic/Action.js

```diff
@@ -0,0 +1,116 @@
+import React, {useContext} from 'react';
+import Select from '../../forms/Select';
+import {ACTION_TYPES, ACTIONS_WITH_OPTIONS, MODIFIABLE_PROPERTIES, PROPERTY_VALUES} from './constants';
+import ComponentSelection from './ComponentSelection';
+import LiteralValueInput from './LiteralValueInput';
+import {ComponentsContext} from './Context';
+import OperandTypeSelection from './OperandTypeSelection';
+import FAIcon from "../../FAIcon";
```

Comment:
```suggestion
import FAIcon from '../../FAIcon';
```
argh, we really should fix them all so that we can copy-paste without worry :)